### DISCUSSION TITLE
about SW_AGENT_COLLECTOR_BACKEND_SERVICES

### DISCUSSION THREAD

**Discussion by zjj1002** [apache/skywalking#11213] [Q&A] [answered]
Hi Community :

1.   about  ”SW_AGENT_COLLECTOR_BACKEND_SERVICES “ 
Can I write both the OAP address and the OTEL collector address here?
Because I want two copies of agent data for OAP and for jaeger
2. May I ask if there is a way to have  another copy of the skywalking Java agent data to opentelemetry collector?
 I want use opentelemetry java agent and exporter to skywalking oap ,but the exporter is still in beta and don't have tracing data.
Thanks!

**Comment by wu-sheng** [comment]
No, you can't. We don't support jeager as backend.

**Comment by zjj1002** [comment]
I just want a data copy  to the opentelemetry collector

**Reply by wu-sheng** [reply, marked as answer]
I just simply give an answer.
No, you can't There is no IF.
